app/assets/stylesheets/components/forms.scss

```diff
@@ -170,6 +172,25 @@ form.thredded--form input[type="text"], form.thredded--form textarea {
   color: $forms-placeholder-text;
 }
 
+.field_with_errors input {
+ &::-webkit-input-placeholder {
```

Comment:
Is there a reason we aren't using https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder ?

Reply:
Good point, I'm gonna remove the browse specific rules.